go/cmd/goserver/main.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"fmt"
+	"github.com/feast-dev/feast/go/internal/feast"
+	"github.com/feast-dev/feast/go/protos/feast/serving"
+	"github.com/kelseyhightower/envconfig"
+	"google.golang.org/grpc"
+	"log"
+	"net"
+)
+
+const (
+	flagFeastRepoPath   = "FEAST_REPO_PATH"
+	flagFeastRepoConfig = "FEAST_REPO_CONFIG"
+	feastServerVersion  = "0.18.0"
+)
+
+type FeastEnvConfig struct {
+	RepoPath   string `envconfig:"FEAST_REPO_PATH"`
```

Comment:
Can we please add comments here explaining exactly what these env vars are used for. Also, is there a reason to favor env vars over CLI arguments? Just curious about the reasoning.

Reply:
I don't have a good argument for env vars over CLI arguments, but will leave as env vars for now given that no one strongly objects

am also removing this struct since it doesn't serve much use